examples/dihedral/rdf.dat

```diff
@@ -0,0 +1,163 @@
+0.000000E+00 3.246401E+01
```

Comment:
add the rdf's as a agr file as in bend?

Reply:
OK, I take this back as xmgrace is likely a dinosaur. I suggest to rename the rdf's to `reference-md-rdf.dat`; `reference-mc-rdf.dat`; and `reference-plot.py`. Instead of plotting to a window, it might be safer to plot to a file (png?)

Reply:
these files should be added to install target as well

Reply:
Updated in 8ff5d87, but needs more attention.